src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -694,6 +697,18 @@ protected function castAttributeAsJson($key, $value)
         return $value;
     }
 
+    /**
+     * Decode the given array into ArrayObject.
+     *
+     * @param array $value
```

Comment:
You should to fix the method docblock before it's being to merged.

Reply:
Whats the problem?

Reply:
```php
/**
 * Decode the given array into ArrayObject.
 *
 * @param  array|object  $value
 * @return \ArrayObject
 */
```

The method accepts `array` and `object` values and never returns `null`.

Reply:
I don't know why it passed the styleci check. Check [Laravel phpDoc] for more details!

Reply:
Done